### ISSUE TITLE
The app enables dangerous file access via setAllowUniversalAccessFromFileURLs

### ISSUE THREAD

**Issue by SachinTichkule** [gree/unity-webview#1043]
net.gree.unitywebview.CWebViewPlugin$4
Line 738 in net/gree/unitywebview/

**Comment by SachinTichkule** [comment]
Use androidx#webkit#WebViewAssetLoader to load file content securely.

**Comment by KojiNakamaru** [comment]
Yes, that part was originally introduced with trusted origins in mind and should be fixed in the future. If you need to avoid it immediately, you can modify the code and build your custom plugin.
cf. https://github.com/gree/unity-webview/tree/ffe3a1748959175ce0f8fa448191cb2738fbc1e0#how-to-build-webviewplugin-aartmpl
